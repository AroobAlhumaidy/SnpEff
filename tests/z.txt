Y	16168170	*	-G/-G	+	0	0	FRAME_SHIFT
