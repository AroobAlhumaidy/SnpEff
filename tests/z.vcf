22	20710051	.	ACGCCGTCCAGGGCGTCGCTAACGAGGTC	AT	.	.	CSQ=T|ENSG00000188280|ENST00000583722|Transcript|downstream_gene_variant|||||||3267|1|FAM230A|HGNC||protein_coding||||||||||||||||||,T|ENSG00000161133|ENST00000454608|Transcript|intron_variant&feature_truncation||||||||-1|USP41|HGNC||protein_coding||||ENST00000454608.2:c.*3-4858_*3-4831delGACCTCGTTAGCGACGCCCTGGACGGCGinsA||||||||||||||,T|ENSG00000161133|ENST00000486536|Transcript|intron_variant&nc_transcript_variant&feature_truncation||||||||-1|USP41|HGNC|20070|processed_transcript||||ENST00000486536.2:n.835-4858_835-4831delGACCTCGTTAGCGACGCCCTGGACGGCGinsA||||||||||||||,T|ENSG00000188280|ENST00000434783|Transcript|feature_truncation|1968-1995|1784-1811|595-604|TPSRASLTRS/M|aCGCCGTCCAGGGCGTCGCTAACGAGGTCg/aTg|||1|FAM230A|HGNC|45045|protein_coding|||PROSITE_profiles:PS50323&PROSITE_profiles:PS50325&Superfamily_domains:SSF56112|ENST00000434783.3:c.1784_1811delCGCCGTCCAGGGCGTCGCTAACGAGGTCinsT|ENSP00000463576.1:p.Thr595_Ser604delinsMet|||||||||||||,T||ENSR00001531265|RegulatoryFeature|regulatory_region_variant||||||||||||||||||||||||||||||
