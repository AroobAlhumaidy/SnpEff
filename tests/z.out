Y	16168170	*	-GGG/-GGG	+	0	0	START_LOST(M/-)
